### PR TITLE
gofmt fix Error string should not be capitalized or end with punctuat…

### PR DESCRIPTION
**Pull request by googs1025**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
Modify the code case to make the code conform to gofmt.
Reference documentation: [here](https://github.com/golang/go/wiki/CodeReviewComments#error-strings)
#### Which issue(s) this PR fixes:

Fixes # None

#### Does this PR introduce a user-facing change?

```release-note
None
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs
None
```

### CODE REVIEW

**Review by swatisehgal — COMMENTED** [review]
Was this issue identified by running a pre-existing script or by inspection? I am asking because I am wondering if this the only location where we have found this issue in the codebase or we need to scrub through the codebase for more such issues at other places too?

File: staging/src/k8s.io/cli-runtime/pkg/resource/helper.go [hunk 1]
```diff
@@ -155,7 +155,7 @@ func EnhanceListError(err error, opts metav1.ListOptions, subj string) error {
 		if len(opts.LabelSelector) == 0 && len(opts.FieldSelector) == 0 {
 			return fmt.Errorf("Unable to list %q: %v", subj, err)
```
Comment (brianpursley) [inline staging/src/k8s.io/cli-runtime/pkg/resource/helper.go:156]:
```suggestion
			return fmt.Errorf("unable to list %q: %v", subj, err)
```

**Pull request closed without merging**